src/script/components/Conversation/Conversation.tsx

```diff
@@ -119,12 +135,104 @@ export const Conversation: FC<ConversationProps> = ({
     }
   }, [readMessagesBuffer.length]);
 
-  if (!mainViewModel) {
-    return null;
-  }
+  const uploadImages = useCallback(
+    (images: File[]) => {
+      if (!activeConversation) {
```

Comment:
can we combine them in one if with || ?

Reply:
if (!activeConversation || isHittingUploadLimit(images, repositories.asset)) {
    return;
}

Reply:
sure 😄